pkgs/development/libraries/zydis/default.nix

```diff
@@ -13,33 +13,25 @@ let
 in
 stdenv.mkDerivation rec {
   pname = "zydis";
-  version = "4.0.0";
+  version = "4.1.0";
 
   src = fetchFromGitHub {
     owner = "zyantific";
     repo = "zydis";
     rev = "v${version}";
-    hash = "sha256-/no/8FNa5LlwhZMSMao4/cwZk6GlamLjqr+isbh6tEI=";
+    hash = "sha256-akusu0T7q5RX4KGtjRqqOFpW5i9Bd1L4RVZt8Rg3PJY=";
   };
 
   nativeBuildInputs = [ cmake ];
   buildInputs = [ zycore ];
   cmakeFlags = [
-    "-DZYAN_SYSTEM_ZYCORE=ON"
```

Comment:
No longer needed explicitly: Zydis now falls back to searching for a system-installed Zycore automatically if the submodule isn't available ([commit](https://github.com/zyantific/zydis/commit/49fa4db3f585a298041b56c15e677bb2e6d0810c)).